Docs/topics/vita.md

```diff
@@ -1,96 +1,69 @@
-On this new empty line, put an asterisk followed by **your** title ID, this should look something like `*PCSC00013`. 
-After you have placed that, make a new line again, and put the following text: `ur0:tai/Allefresher_vita.suprx`
+### Patching LBP Vita
 
-After you are done, you should see something like this, but with your title ID.
+Navigate to the installation menu by selecting `Vita plugins` → `Miscellaneous Plugins` → `LittleBigPlanet Custom Server`.
 
-![2024-05-04-210838.png](vitashell-after-config-edit.png)
+On this menu, you should see every installed copy of LittleBigPlanet Vita. 
+Select the one you want to patch. 
+It will install the plugin for that specific copy only. 
+Afterward, a green dot should appear at the right of the screen corresponding to the game you patched.
+If you don't see your game on this list, make sure updates are installed, then try again.
 
-<warning>
-Make sure that you do not place the plugin's name under <code>*main</code> or <code>*KERNEL</code>, this could cause your Vita to not turn on. 
-If this happens, hold the <code>L</code> button on boot to disable any plugins, then fix the config in VitaShell.
-</warning>
+### Patching LBP PSP
 
-### Changing the server you are connecting to
+Install the plugin by selecting `PSP plugins` → `PSP plugins for Adrenaline` → `Allefresher by Beyley`.
 
-If you are connecting to the official LittleBigRefresh instance, you can ignore this segment, and skip to [Step 5](#step-5-finalize-the-setup).
+## Step 4: Finalize the setup!
 
-Allefresher Vita allows the user to specify another server to connect to, although it requires a bit more setup. 
+Exit out of AutoPlugin2 editor by pressing the correct back button for your region, then selecting `Exit` on AutoPlugin2's menu.
+Finally, restart your Vita.
 
-Allefresher will read from a text file named `allefresher.txt` from `ux0:` on your Vita to get the URL to connect with, 
-defaulting to the LittleBigRefresh instance if not found. 
-This file contains the patching URL for the server you want to connect to. You can get this URL from the server owner.
+Now, ensure your game is fully up to date. 
 
-However, for the game to be able to read this file, you need to install the [ioPlus](https://www.gamebrew.org/wiki/IoPlus_Vita) plugin.
-This is because the Vita disallows the game to access files outside its own directory, and Allefresher has the same permissions as the game itself.
-ioPlus expands the permissions of the game to allow read/write to arbitrary files, allowing Allefresher to find your config.
+### Updating LBP Vita
 
-## Step 5: Finalize the setup!
+Press triangle twice with the game's bubble selected in the Live Area. 
+If the Update History button is present and shows Version 1.22, then your game is fully up to date. 
+If it is not, open the game's live area, and press the refresh button at the top of the screen, this will install all the updates for your game.
```

Comment:
When pressing the refresh button, a yellow arrow button should appear. Press the yellow arrow button to install all the updates for your game.